src/geometry/path2/close.js

```diff
@@ -0,0 +1,34 @@
+const {EPS} = require('../../core/constants')
+
+const vec2 = require('../../math/vec2')
+
+const clone = require('./clone')
+
+/**
+ * Close the given geometry.
+ * @params {geometry} the path to close
+ * @returns {path} the closed path
+ */
+const close = (geometry) => {
```

Comment:
sorry If I am being dense here, but why do we need so much complexity ?
Can a closed path not be defined just as having the same start & end point ?
And would closing a path not be just appending the point if needed ?